app/src/main/java/org/mozilla/fenix/exceptions/ExceptionsFragment.kt

```diff
@@ -92,8 +96,7 @@ class ExceptionsFragment : Fragment() {
 
     private fun reloadExceptions() {
         trackingProtectionUseCases.fetchExceptions { resultList ->
-            val exceptionsList = resultList.map { ExceptionsItem(it) }
```

Comment:
Instead of making all of the changes in this PR I think we could just change this to

```
val exceptionsList resultList.map { ExceptionsItem(it.url) }
```